docs/src/examples/components/Portal/Types/PortalExample.shorthand.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Button, Divider, Header, Label, Portal } from '@stardust-ui/react'
+
+class PortalExamplePortal extends React.Component<any, any> {
+  public state = {
+    log: [],
+    logCount: 0,
+  }
+
+  public render() {
```

Comment:
I don't think we should be typing public/private like this in React components.  Even though `method` and other custom methods are publicly accessible, it is not idiomatic React to actually access them.  You should never reach into a React component and call a method.

Reply:
I don't agree with this and here are my reasons:

1. The current pattern I see all over the project is to not specify any modifier for class members which means all members are public (class members are public by default in typescript).
This is wrong and is one of the main reasons I'd recommend enforcing modifiers; developers often forget to specify a method is private since no modifier is required and this makes it private.

2. `render` is a public method as it comes from `React.Component` or `UIComponent`, the classes we usually extend.

3. Specifying modifiers makes it clear for developers coming from other environments where `private` is the default class member modifier (such as `C#`).

**We had this discussion before and since it's occurring again I can create an RFC for enforcing modifiers if you think it's valuable.**

@levithomason @kuzhelov @miroslavstastny @mnajdova what do you guys think?

Reply:
since this is an example I'll remove the modifiers so that we make it `jsx`, but the points stand for actual typescript code